crates/bdk/src/wallet/mod.rs

```diff
@@ -527,11 +527,12 @@ impl<D> Wallet<D> {
 
     /// Return the balance, separated into available, trusted-pending, untrusted-pending and immature
     /// values.
-    pub fn get_balance(&self) -> Balance {
-        self.keychain_tracker.balance(|keychain| match keychain {
-            KeychainKind::External => false,
-            KeychainKind::Internal => true,
-        })
+    pub fn get_balance(&self, include_reserved_utxos: bool) -> Balance {
```

Comment:
Similarly to above, when users ask for a balance I suppose they always want the reserved utxos anyways